javascript/javascript-core/functions-i/setting-default-values-with-short-circuiting.md

```diff
@@ -75,11 +75,18 @@ function title(t) {
 
 ## Practice
 
-"Short circuit" the following code:
+Complete both functions. 
+
+When `name` is undefined, `shortCirA` and `shortCirB` should return `'John Doe'` and `'Jane Doe'` respectively. 
+
+If `name` is defined, then each function should return the name:
 
 ```javascript
-function shortCir(name){
-  return ??? ??? 'John Doe';
+function shortCirA(name){
+  return ??? || 'John Doe';
+}
+function shortCirB(name){
+  return name ??? 'Jane Doe';
 }
```

Comment:
What do you think of making the question testing both old (via ||) and new (via ??) way of short circuiting:

````md
Complete both functions:

1. When `name` is any falsy value, `shortCirA` should return `'John Doe'`.

2. When `name` is `undefined` or `null`, `shortCirB` should return `'Jane Doe'`. 

```javascript
function shortCirA(name){
  return name ??? 'John Doe';
}

function shortCirB(name){
  return name ??? 'Jane Doe';
}
```

- ||
- ??
- &&
- ==
````

Reply:
Ya I like that a lot -- Will add